pkgs/data/fonts/glasstty-ttf/default.nix

```diff
@@ -0,0 +1,18 @@
+{ runCommand, fetchurl, lib }:
+runCommand "glasstty-ttf" {
```

Comment:
Why is this in a runCommand and not in the postFetch of fetchurl?

Reply:
https://github.com/NixOS/nixpkgs/pull/192787#discussion_r979274422